language/sentiment/README.txt

```diff
@@ -0,0 +1,27 @@
+# Introduction
+
+This sample contains the code referenced in the 
+[Sentiment Analysis Tutorial](http://cloud.google.com/natural-language/docs/sentiment-tutorial)
+within the Natural Language API Documentation. A full walkthrough of this sample
+is located within the documentation.
+
+This sample is a simple illustration of how to construct a sentiment analysis
+request and process a response using the API.
+
+## Download the Code
+
+```
+git clone sso://python-dev-samples/language/sentiment/
+```
+
+### Usage
```

Comment:
Should probably add a section for creating a virtualenv and installing the requirements. [example](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/speech/api-client#setup)